### DISCUSSION TITLE
Sizing to deploy on openshift o kubernetes

### DISCUSSION THREAD

**Discussion by Dasaenz499** [arkmq-org/arkmq-org-broker-operator#593] [General]
Hi, any recommendations to do a sizing of artemis in kubernetes platforms. Maybe if there is a excel o any tool for this.... thanks

**Comment by gaohoward** [comment]
Hi can you give some specifics about what `sizing` means? Do you mean performance in any aspect of artemis?

**Comment by brusdev** [comment]
Closed because no updates in 2 weeks.
